### ISSUE TITLE
Feature Request: change map projection to round globus (like Google Maps does)

### ISSUE THREAD

**Issue by erik55** [osmandapp/Osmand#5889]
Google Maps has changed its projection from [Mercator projection](https://en.wikipedia.org/wiki/Mercator_projection) to a round globus. Why don’t we do that for Osmand? It is very nice to have a good comparison of country sizes and a feeling of “we are living on a marble”.

**Comment by daniel-callejas-sevilla** [comment]
This is especially interesting when using the radius ruler at low levels of zoom. For example, in the screenshot below, the radius ruler says I'm between 1500 km and 2000 km, while the actual distance calculated is 2328 km. That error is due to the Mercator projection.
![6587](https://user-images.githubusercontent.com/10685659/72686870-1d3c9a00-3af9-11ea-86a6-0679d98fda48.png)

**Comment by erik55** [comment]
The **Magic Earth** app also uses a sphere on low zoom levels. But the projection change from globe to Mercator looks a bit funny/abrupt. Should be done better in Osmand.

**Comment by vshcherb** [comment]
Mercator project is very good for navigation, it persuades simple idea that you can use in navigation - [rhumb](https://en.wikipedia.org/wiki/Rhumb_line)

**Comment by navid-zamani** [comment]
Yeah, so very good that it tells you you’re between 1500 and 2000 km away, when you’re really 2328km away. ;)
So good it tells you Greenland is as big and hence takes as long to cross as Africa. ^^

What good is simplicity when it limits you? It’s not like 3 dimensions is somehow mentally overwhelming. ^^
Earth is a 3-dimensional object. In the 21st century, a 2D projection does nothing but limit. Let’s not limit the users. We’re not using paper maps anymore.

(That being said, obviously nobody is obliged to actually code it, and not willing to do that work if of course always justified.)

**Comment by dmpr0** [comment]
Globus projection is implemented and will be available in the 5.3 release. It can be enabled in the Menu – Configure Map – 
Globe view.

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/fc8a158b-d749-47fa-8432-be7e05d71ab2"  width="300" height="auto"/> </td>
  </tr>
</table>
